src/store/callRecordingStore.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @copyright Copyright (c) 2022 Marco Ambrosini <marcoambrosini@icloud.com>
+ *
+ * @author Marco Ambrosini <marcoambrosini@icloud.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { startRecording, stopRecording } from '../services/callsService.js'
+import { set } from 'Vue'
+
+const state = () => ({
+	isRecording: false,
```

Comment:
@marcoambrosini you remember why you went for a dedicated callRecordingStore, instead of relying on the property of the conversation object?
I assume because it did not exist back then?